client/tx/search.go

```diff
@@ -120,6 +134,28 @@ func searchTxs(cliCtx context.CLIContext, cdc *codec.Codec, tags []string) ([]In
 	return info, nil
 }
 
+func getKeyFromBechPrefix(key string, value string) (string, string, string) {
```

Comment:
I think this function should be renamed to something more generic along the lines of `parseTagKeyValue`.

Reply:
deleted function (see comment below)